bot/jobs.py

```diff
@@ -287,3 +288,63 @@ def check_versions_status(context):
                           f"but one of the nodes already runs on *{highest_version}*"
                 try_message_with_home_menu(context, chat_id=context.job.context['chat_id'],
                                            text=message)
+
+def check_churning(context):
+    user_data = context.job.context['user_data']
+    service = LocalStorageService(context)
+
+    try:
+        validators = get_node_accounts()
+    except Exception as e:
+        logger.exception(e)
+        message = 'I couldn\'t check the statuses of other nodes in network! List of nodes currently unavailable!'
+        try_message_with_home_menu(context, chat_id=context.job.context['chat_id'], text=message)
+        return
+
+
+    if 'node_statuses' not in user_data:
+        user_data['node_statuses'] = {}
+        for validator in validators:
+            user_data['node_statuses'][validator['node_address']] = validator['status']
+        return
+
+    local_node_statuses = service.get_node_statuses()
+
+    churned_in = []
+    churned_out = []
+    for validator in validators:
+        remote_status = validator['status']
+        local_status = local_node_statuses[validator['node_address']]
+        if remote_status != local_status:
+            if 'active' == remote_status:
+                churned_in.append({"address": validator['node_address'], "bond": validator['bond']})
+            elif 'active' == local_status:
+                churned_out.append({"address": validator['node_address'], "bond": validator['bond']})
+
+    if len(churned_in) or len(churned_out):
+        text = "🔄 CHURN SUMMARY\n" \
+               "THORChain has successfully churned:\n\n" \
+               "Nodes Added:\n"
+        for node in churned_in:
+            text += "*" + node['address'] + "*\nBond: *" + tor_to_rune(node['bond']) + '*\n'
```

Comment:
```suggestion
            text += f"*{node['address']}*\nBond: *{tor_to_rune(node['bond'])}*\n"
```
i found that from python 3.6 we can use string interpolation like this. Doesn't it look cleaner?